tests/templates/y_cable_simulator_client.j2

```diff
@@ -269,3 +277,125 @@ def check_if_link_is_active_for_torB(physical_port):
     """
     return True
 
+def enable_prbs_mode(physical_port, target, mode_value, lane_map):
```

Comment:
i would suggest putting a generic note for all these dummy values, so as to ensure that this needs to be corrected in future. suggest putting a TODO as well.

Reply:
What needs to be done in the future? Since these hardware constructs don't exist for the simulator shouldn't we always use these dummy values?

Reply:
If we enable /disable loopback or prbs then normal traffic cannot run. I dont know how exactly you can accomplish this with simulator.

Reply:
Is this something that would be used during normal operation?